server/@types/dto/index.d.ts

```diff
@@ -60,7 +60,7 @@ declare module 'inductionDto' {
 
   export interface WorkOnReleaseDto extends ReferencedAndAuditable {
     hopingToWork: HopingToGetWorkValue
-    notHopingToWorkReasons: ReasonToNotGetWorkValue[]
+    notHopingToWorkReasons: ReasonNotToGetWorkValue[]
```

Comment:
Previously we had a mix of `reason(s)NotTo` and `reason(s)ToNot`, but I'm consistently using the former (except for `notHopingToWorkReasons` here, as that models the API, rather than the UI codebase).